api/KeyboardEvent.json

```
@@ -1101,7 +1101,7 @@
             "status": {
               "experimental": false,
               "standard_track": true,
```

suggestion: reflect the standards status, the opposite of the others in this PR

```suggestion
              "standard_track": false,
```